server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -253,7 +260,7 @@ module.exports = {
           href: '/hdc/victim/victimLiaison/',
           text: 'View/Edit',
         },
-        visible: eligible && (curfewAddressApproved || bassOfferMade),
+        visible: eligible && validAddress,
```

Comment:
`eligible && validAddress` is used in several places probably worth having a variable?

Reply:
As above, have split out the small sub-set of tasks that apply when ineligible, then the conditions in the "eligble" task list are cleaner